### ISSUE TITLE
Loss functions

### ISSUE THREAD

**Issue by JT4v4res** [JT4v4res/Neural-networks-and-deep-learning#33]
Loss functions needs to be implemented so we can use them for the backpropagation calculation.
